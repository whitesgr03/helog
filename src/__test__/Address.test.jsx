import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";

import Address from "../components/Address";

describe("Address component", () => {
	it("should render correctly", () => {
		const { asFragment } = render(<Address />);

		const actual = asFragment();

		expect(actual).toMatchSnapshot();
	});
});
