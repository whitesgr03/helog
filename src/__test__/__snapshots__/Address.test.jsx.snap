// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Address component > should render correctly 1`] = `
<DocumentFragment>
  <address>
    <a
      href="mailto:whitesgr03@gmail.com"
    >
      <span
        class="_icon_7e06e0 _email_0a131a"
      />
      <em>
        whitesgr03@gmail.com
      </em>
    </a>
  </address>
</DocumentFragment>
`;
